src/ui/reducers/app.ts

```diff
@@ -370,3 +370,18 @@ export const isCurrentTimeInLoadedRegion = createSelector(
     );
   }
 );
+
+export const isPointInLoadedRegion = createSelector(
+  getLoadedRegions,
+  (state: UIState, executionPoint: string) => executionPoint,
+  (regions: LoadedRegions | null, executionPoint: string) => {
+    return (
+      regions !== null &&
+      regions.loaded.some(
```

Comment:
I think we actually care about `loading` here, not `loaded`. I'm fairly sure that f we request a pause at a time where the backend is still loading it's fine, it will just take a while to get created.